common/locales/en/settings.json

```diff
@@ -126,6 +128,7 @@
     "unsubscribedTextOthers": "You won't receive any other email from Habitica.",
     "unsubscribeAllEmails": "Check to Unsubscribe from Emails",
     "unsubscribeAllEmailsText": "By checking this box, I certify that I understand that by unsubscribing from all emails, Habitica will never be able to notify me via email about important changes to the site or my account.",
+    "unsubscribeAllPush": "Check to Unsubscribe from all Push Notifications",
```

Comment:
The capitalization is the same as with the unsubscribeAllEmails string. So it should probably be also changed there, if it's changed here?